# [[코드업] Python 기초 100제](https://codeup.kr/problemsetsol.php?psid=33)

#### 6006

In [7]:
print('\"!@#$%^&*()\'')

"!@#$%^&*()'


#### 6007

"C:\Download\'hello'.py"  
(단, 따옴표도 함께 출력한다.)

In [13]:
print("\"C:\Download\\'hello'.py\"")

"C:\Download\'hello'.py"


#### 6008

print("Hello\nWorld")  
위 코드를 정확히 그대로 출력하시오.

In [15]:
print('print("Hello\\nWorld")')

print("Hello\nWorld")


#### 6009

In [17]:
c = input()
print(c)

 ed


ed


#### 6010

In [18]:
c = input()
c = int(c)
print(c)

 -1


-1


#### 6012

In [19]:
a = input()
b = input()
print(a)
print(b)

 2
 3


2
3


#### 6013

In [21]:
a = input()
b = input()
print(b)
print(a)

 a
 b


b
a


#### 6014

In [22]:
a = float(input())
print(a)
print(a)
print(a)

 3.14


3.14
3.14
3.14


#### 6015
공백을 두고 입력된정수(integer) 2개를 입력받아 줄을 바꿔 출력해보자.

In [27]:
a, b = input().split()
print(a)
print(b)

 a 2


a
2


#### 6016
공백을 두고 문자(character) 2개를 입력받아 순서를 바꿔 출력해보자.

In [32]:
a, b = str(input()).split()
print(b)
print(a)

 qkr dud


dud
qkr


#### 6017
정수(integer), 실수, 문자(character), 문자열(string) 등 1개만 입력받아 한 줄로 3번 출력해보자.

In [37]:
c = input()
print(c, c, c)

 1


1 1 1


#### 6018
24시간 **시:분 형식으로 시간이 입력**될 때, 그대로 출력하는 연습을 해보자.

In [41]:
a, b = input().split(':')
print(a, b, sep = ':')

 3:16


3:16


#### 6019

"연도.월.일"을 입력받아 "일-월-연도" 순서로 바꿔 출력해보자.

In [43]:
a, b, c = input().split('.')
print(c, b, a, sep='-')

 2021.02.04


04-02-2021


#### *6020*
주민번호를 입력받아 형태를 바꿔 출력해보자.  
'-'를 제외한 주민번호 13자리를 모두 붙여 출력한다.

In [45]:
a, b = input().split('-')
print(a, b, sep="")

 000907-1121112


0009071121112


#### 6021
알파벳과 숫자로 이루어진 단어 1개가 입력된다.  
입력받은 단어의 각 문자를 한 줄에 한 문자씩 분리해 출력한다.

In [50]:
a = str(input())
for i in range(len(a)):
    print(a[i])

 Hello


H
e
l
l
o


#### 6022
6자리의 연월일(YYMMDD)을 입력받아 나누어 출력해보자.  
년도(YY) 월(MM) 일(DD)을 공백으로 구분해 한 줄로 출력한다.

In [53]:
a = input()
print(a[0:2], a[2:4], a[4:6], sep=" ")

 971219


97 12 19


#### 6023
시:분:초 형식으로 시간이 입력될 때 분만 출력해보자.

In [58]:
a=input().split(':')
print(a[1])

 17:23:57


23


#### 6024
알파벳 문자와 숫자로 이루어진 단어 2개를 입력받아  
순서대로 붙여 출력하는 프로그램을 작성해보자.

In [59]:
a, b = input().split(' ')
print(a, b, sep='')

 hello world


helloworld


#### 6025
정수 2개를 입력받아 합을 출력하는 프로그램을 작성해보자.

In [64]:
a, b = input().split(' ')
print(int(a) + int(b))

 123 -123


0


#### 6026
실수 2개를 입력받아 합을 출력하는 프로그램을 작성해보자.

In [67]:
a = input()
b = input()
c = float(a) + float(b)
print(c)

 0.1
 0.9


1.0


#### *6027*
10진수를 입력받아 16진수(hexadecimal)로 출력해보자.

In [70]:
a = input()
n = int(a)
print('%x'%n) # 16진수(hexadecimal)

 255


ff


#### *6028*
10진수를 입력받아 16진수(hexadecimal)로 출력해보자.  
print('%X' % n)  #n에 저장되어있는 값을 16진수 **대문자** 형태 문자열로 출력

In [5]:
a = input()
n = int(a)
print('%X'%n)

 255


FF


#### *6029*
16진수를 입력받아 8진수(octal)로 출력해보자.

In [6]:
a = input()
n = int(a, 16)
print('%o'%n) # 8진주(octal)

 f


17


#### *6030*
영문자 1개 입력받아 10진수로 변환하기

In [7]:
a = ord(input())
print(a)

 A


65


#### *6031*
10진 정수 1개를 입력받아 유니코드 문자로 출력해보자.

In [10]:
a = int(input())
print(chr(a))

 65


A


#### *6033*
문자 1개를 입력받아 그 다음 문자를 출력해보자.  
영문자 'A'의 다음 문자는 'B'이고, 숫자 '0'의 다음 문자는 '1'이다.

In [15]:
a = input()
b = ord(a)
print(chr(b + 1))

 a


b


#### 6036
단어와 반복 횟수를 입력받아 여러 번 출력해보자.

In [18]:
a, b = input().split()
print(a*int(b))

 love 3


lovelovelove


#### 6037
반복 횟수와 문장을 입력받아 여러 번 출력해보자.

In [19]:
a = int(input())
b = input()
print(a*b)

 3
 I love CS


I love CSI love CSI love CS


#### 6038
정수 2개(a, b)를 입력받아
a를 b번 곱한 거듭제곱을 출력하는 프로그램을 작성해보자.

In [26]:
a, b = input().split()
a = int(a)
b = int(b)
print(a**b)

 2 10


1024


#### 6039
실수 2개(f1, f2)를 입력받아 f1을 f2번 거듭제곱한 값을 출력하는 프로그램을 작성해보자.

In [27]:
a, b = input().split()
a = float(a)
b = float(b)
print(a**b)

 4.0 2.0


16.0


#### 6040
a를 b로 나눈 몫을 출력한다.

In [32]:
a, b = input().split()
a = int(a)
b = int(b)
print(a//b)

 10 3


3


#### 6042
실수 1개를 입력받아 소숫점 이하 두 번째 자리까지의 정확도로 반올림한 값을 출력해보자.

In [34]:
a = float(input())
print('{:.2f}'.format(a))

 3.141592


3.14


In [35]:
a=float(input())
print( format(a, ".2f") )

 3.141592


3.14


#### 6044

정수 2개(a, b)를 입력받아 합, 차, 곱, 몫, 나머지, 나눈 값을 자동으로 계산해보자.  
단 0 <= a, b <= 2147483647, b는 0이 아니다.

In [44]:
a, b = input().split()
a = int(a); b = int(b)
print(a+b)
print(a-b)
print(a*b)
print(a//b)
print(a%b)
print(format(a/b, '.2f'))

 10 3


13
7
30
3
1
3.33


#### 6045
정수 3개를 입력받아 합과 평균을 출력해보자.

In [49]:
a, b, c = input().split()
s = int(a) + int(b) + int(c)
m = ( int(a) + int(b) + int(c) ) / 3
print(s, format(m, '.2f'))

 1 2 3


6 2.00


#### *6046*
정수 1개를 입력받아 2배 곱해 출력해보자.

In [51]:
a = int(input())
print(a<<1)

 1024


2048


#### *6047*
정수 2개(a, b)를 입력받아 a를 2^b배 곱한 값으로 출력해보자.  
0 <= a <= 10, 0 <= b <= 10

In [58]:
a, b = map(int, input().split())
print(a<<b)

 1 3


8


#### 6048
두 정수(a, b)를 입력받아 a가 b보다 작으면 True 를, a가 b보다 크거나 같으면 False 를 출력하는 프로그램을 작성해보자.

In [63]:
a, b = map(int, input().split())
if a < b:
    print('True')
else:
    print('False')

 1 9


True


#### *6052*
정수가 입력되었을 때, True/False 로 평가해주는 프로그램을 작성해보자.

In [68]:
a= int(input())
print(bool(a))

 0


False


#### *6053*
정수값이 입력될 때, 그 불 값을 반대로 출력하는 프로그램을 작성해보자.

In [72]:
a= bool(int(input()))
print(not a)

 1


False


#### 6054
2개의 정수값이 입력될 때, 그 불 값이 모두 True 일 때에만 True 를 출력하는 프로그램을 작성해보자.

In [75]:
a, b = map(int, input().split())
if bool(a) & bool(b):
    print('True')
else:
    print('False')

 0 1


False


#### *6056*
2개의 정수값이 입력될 때, 그 불 값(True/False) 이 서로 다를 때에만 True 를 출력하는 프로그램을 작성해보자.


In [84]:
a, b = map(int, input().split())
c = bool(a)
d = bool(b)
print((c and (not d)) or ((not c) and d))

 1 1


False


#### *6057*
2개의 정수값이 입력될 때, 그 불 값(True/False) 이 서로 같을 때에만 True 를 출력하는 프로그램을 작성해보자.

In [90]:
a, b = map(int, input().split())
c = bool(a)
d = bool(b)
print((c and d) or ((not c) and (not d)))

 0 0


True


#### 6058
2개의 정수값이 입력될 때, 그 불 값(True/False) 이 모두 False 일 때에만 True 를 출력하는 프로그램을 작성해보자.

In [91]:
a, b = map(int, input().split())
c = bool(a)
d = bool(b)
print((not c and not d))

 0 0


True


#### *6063*
입력된 두 정수(a, b) 중 큰 값을 출력하는 프로그램을 작성해보자. 단, 3항 연산을 사용한다.

In [92]:
a, b = map(int, input().split())
c = (a if a>=b else b)
print(int(c))

 123 456


456


#### 6064
입력된 세 정수 a, b, c 중 가장 작은 값을 출력하는 프로그램을 작성해보자. 단, 3항 연산을 사용한다.

In [94]:
a, b, c = map(int, input().split())
m = (b if a>b else a)
n = (m if m<c else c)
print(int(n))

 3 0 -2


-2


#### *6065*
3개의 정수(a, b, c)가 입력되었을 때, 짝수만 출력해보자.

In [97]:
a, b, c = map(int, input().split())
if a%2== 0:
    print(a)
if b%2==0:
    print(b)
if c%2==0:
    print(c)

 1 2 4


2
4


#### 6067
0이 아닌 정수 1개가 입력되었을 때, 음(-)/양(+)과 짝(even)/홀(odd)을 구분해 분류해보자.  
음수이면서 짝수이면, A  
음수이면서 홀수이면, B  
양수이면서 짝수이면, C  
양수이면서 홀수이면, D  
를 출력한다.

In [98]:
a = int(input())
if a<0 and a%2 == 0:
    print('A')
elif a<0 and a%2==1:
    print('B')
elif a>0 and a%2 == 0:
    print('C')
else:
    print('D')

 -2147483648


A


#### 6070
월이 입력될 때 계절 이름이 출력되도록 해보자.

월   :    계절 이름  
12, 1, 2 : winter  
  3, 4, 5 : spring  
  6, 7, 8 : summer  
  9, 10, 11 : fall  

In [103]:
a = int(input())
if a in (12, 1, 2):
    print('winter')
elif a in (3, 4, 5):
    print('spring')
elif a in (6, 7, 8):
    print('summar')
else:
    print('fall')

 7


summar


#### *6071*
임의의 정수가 줄을 바꿔 계속 입력된다.
-2147483648 ~ +2147483647, 단 개수는 알 수 없다.

0이 아니면 입력된 정수를 출력하고, 0이 입력되면 출력을 중단해보자.

In [109]:
while True: 
    a=int(input())
    if a==0: 
        break 
    else: 
        print(a)

 7


7


 4


4


 2


2


 3


3


 0


#### 6072
정수(1 ~ 100) 1개가 입력되었을 때 카운트다운을 출력해보자.

In [113]:
a = int(input())
while True:
    print(a)
    a -= 1
    if a==0:
        break

 5


5
4
3
2
1


#### 6073
정수(1 ~ 100) 1개가 입력되었을 때 카운트다운을 출력해보자.

while 조건식 :  
  ...  
  ...  
  
반복 실행구조를 사용해 보자.

In [2]:
a= int(input())
while True:
    a -= 1
    print(a)
    if a==0:
        break

 5


4
3
2
1
0


#### *6074*
영문 소문자(a ~ z) 1개가 입력되었을 때, a부터 그 문자까지의 알파벳을 순서대로 출력해보자.

In [8]:
c = ord(input())
t = ord('a')
while t<=c:
    print(chr(t), end=' ')
    t += 1

 f


a b c d e f 

#### *6075*
정수(0 ~ 100) 1개를 입력받아 0부터 그 수까지 순서대로 출력해보자.

In [ ]:
a = int(input())
while True:
    